### PR TITLE
[MU4] mpe_articulation_profiles_provider

### PR DESCRIPTION
**Pull request by vpereverzev**
Implemented reading and writing methods for MPE Articulation Profiles. The very first step of profile

### CODE REVIEW

File: src/framework/mpe/iarticulationprofilesprovider.h [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+#define MU_MPE_IARTICULATIONPROFILESPROVIDER_H
+
+#include "modularity/imoduleexport.h"
+#include "async/channel.h"
+#include "io/path.h"
+
+#include "mpetypes.h"
+
+namespace mu::mpe {
+class IArticulationProfilesProvider : MODULE_EXPORT_INTERFACE
```
Comment (igorkorsukov) [inline src/framework/mpe/iarticulationprofilesprovider.h:33]: I believe this is a "repository"

Comment (vpereverzev) [reply src/framework/mpe/iarticulationprofilesprovider.h:33]: I'm not against, will rename

Comment (vpereverzev) [reply src/framework/mpe/iarticulationprofilesprovider.h]: done

File: src/framework/mpe/iarticulationprofilesrepository.h [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+#include "modularity/imoduleexport.h"
+#include "async/channel.h"
+#include "io/path.h"
+
+#include "mpetypes.h"
+
+namespace mu::mpe {
+class IArticulationProfilesRepository : MODULE_EXPORT_INTERFACE
+{
+    INTERFACE_ID(IArticulationProfilesProvider)
```
Comment (igorkorsukov) [inline src/framework/mpe/iarticulationprofilesrepository.h:35]: needs change

Comment (vpereverzev) [reply src/framework/mpe/iarticulationprofilesrepository.h]: yeah, just noticed it
fixed

**Review by igorkorsukov — APPROVED** [review]

**Pull request merged**